apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/EarlySecurityChecks/ExitChecksDrawer.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { Button, Flex, IconsLegacy, Link } from "@ledgerhq/react-ui";
+import { useTranslation } from "react-i18next";
+import DrawerFooter from "./DrawerFooter";
+import { withV3StyleProvider } from "~/renderer/styles/StyleProviderV3";
+import ErrorDisplay from "../../../ErrorDisplay";
+import { createCustomErrorClass } from "@ledgerhq/errors";
+import TrackPage from "~/renderer/analytics/TrackPage";
+import { track } from "~/renderer/analytics/segment";
+
+export type Props = {
+  onClose: () => void;
+  onClickExit: () => void;
+};
+
+const ExitError = createCustomErrorClass("ExitEarlySecurityChecksWarning");
```

Comment:
Idk if it's a good idea to create errors like this just to use `ErrorDisplay`. 
We should keep this for errors that are really thrown from somewhere on our libs

If it's coming from the logic of components (from LLM or LLD), and no error was thrown from our libs/hooks, we should just display the contents directly, without creating an error.

WDYT ?